source/extensions/config_subscription/grpc/grpc_mux_impl.cc

```diff
@@ -130,6 +130,15 @@ void GrpcMuxImpl::sendDiscoveryRequest(absl::string_view type_url) {
   }
 }
 
+void GrpcMuxImpl::clearNonce() {
```

Comment:
Makes sense. Have you verified out of the N mux impls that this issues is now consistently handled?

Reply:
The Delta and Unified (both SotW and Delta) already had this fixed.